sdk/eidolon_ai_sdk/bin/agent_http_server.py

```diff
@@ -238,6 +242,7 @@ def start_app(lifespan):
         allow_methods=["*"],
         allow_headers=["*"],
     )
+    _app.add_middleware(LoggingMiddleware)
```

Comment:
moved logging middleware to bottom so it is called first. Missling the logs if another layer errors is not fun